tests/resource/resource.test.js

```diff
@@ -78,7 +78,7 @@ describe('Resource(basic)', function() {
   // TESTS
   describe('construction', function() {
     it('should throw an exception if called with an invalid model', function(done) {
-      expect(rest.resource).to.throw('please specify a valid model');
+      expect(rest.resource.bind(rest)).to.throw('please specify a valid model');
```

Comment:
@mbroadst I had to edit this test because `except` was calling `resource` method without `rest` context. It doesn't change at all the purpose of this test.